xml/System.Security.Cryptography.Pkcs/Rfc3161TimestampTokenInfo.xml

```diff
@@ -328,8 +348,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value indicating if every timestamp token from the same TSA can always be ordered based on the <see cref="P:System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo.Timestamp" />, regardless of the accuracy; If <see langword="false" />, <see cref="P:System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo.Timestamp" /> indicates when the token has been created by the TSA.</summary>
+        <value>A value indicating if every timestamp token from the same TSA can always be ordered based on the <see cref="P:System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo.Timestamp" />.</value>
```

Comment:
```suggestion
        <value><see langword="true"/> if every timestamp token from the same TSA can always be ordered based on the <see cref="P:System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo.Timestamp" />; <see langword="false"/> otherwise.</value>
```